controllers/reports/index.js

```diff
@@ -0,0 +1,111 @@
+import db from '../../db/models';
+
+export default {
+  reportArticle: async (req, res) => {
+    const {
+      user, body: {
+        articleId, message
+      }
+    } = req;
+
```

Comment:
Great work. Can you check if the article exists first before moving on to create a report for the article